src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/BedSearchResultTransformer.kt

```diff
@@ -93,6 +93,7 @@ class BedSearchResultTransformer {
           crn = it.crn,
           days = it.days,
           bookingId = it.bookingId,
+          roomId = it.roomId,
```

Comment:
How come we are now returning the roomId? If we've learnt something new that might be well documented in the commit. This also looks like a good candidate for absorbing back into the original commits, perhaps with a reference to "we didn't think we needed the roomId but it turns out we do beacuse x,y,z"

Reply:
I've squashed and edited this commit to explain the change - the front-end URL for the booking that this would link to is at `/{premisesId}/{roomId}/{bookingId}`, so this field needed to be included. Again, see above - I'd love to absorb this into previous commits, pretty sure it isn't possible on already-commited-and-pushed-things, could cherry-pick but might take time which could be better-spent elsewhere, esp. as this is blocking the front-end work.

Reply:
See above - this is now properly absorbed into previous commits.

Reply:
👍 

[I wondered earlier if the premises id and room id was being pulled from the BedSearchResult ](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/908#discussion_r1300124087), do we not need to do it for premisesId too if it's for building the link?